src/frontend/js/widgets/Dashboard/components/DashboardLayoutRoute/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Outlet, useMatches } from 'react-router-dom';
+import { DashboardRouteHandle } from 'widgets/Dashboard/hooks/useDashboardRouter';
+import { DashboardLayout } from 'widgets/Dashboard/components/DashboardLayout';
+
+export const DashboardLayoutRoute = () => {
+  const matches = useMatches();
+  const renderOutletOnly = matches.find((match) => {
```

Comment:
I understand why this logic is implemented but it feels quite weird imo, let's discuss in daily if we can find a more easy approach 🤔

Reply:
That the point.
We're currently using a parent route element like this : 
```js
<Route path='/dashboard' element={<DashboardLayout />}>
  {/*  DashboardCourses is displayed inside parent route DashboardLayout */}
  <Route path='/dashboard/courses' element={<LearnerDashboardCourses />} />
  {/* ... other learner routes */}
  {/*  dashboard/teacher routes should not be displayed inside parent route DashboardLayout */}
  <Route path='/dashboard/teacher' element={<Outlet />} />
    <Route path='/dashboard/teacher/courses' element={<TeacherDashboardCourses />} />
    {/* ... other teacher routes */}
  </Route>
</Route>
```
To get wanted behavior, i changed `DashboardLayout` by a `DashboardLayoutRoute` that will not display `DashboardLayout` if `route.handle.renderLayout` is set to true.

Reply:
Indeed, if we wanted to avoid this, we'd need to create a dedicated `/students` part which would introduce too much complexity.